app/components/TrackDetails/index.js

```diff
@@ -58,6 +70,15 @@ export function TrackDetails({ collectionName, artistName, collectionPrice, artw
       <If condition={!isEmpty(location.state)} otherwise={<T data-testid="cost-unavaiable" id="cost_unavailable" />}>
         <T data-testid="total-cost" id="total_cost" values={{ price: location.state.item.collectionPrice }} />
       </If>
+      <If condition={!isEmpty(location.state)&& !isEmpty(location.state.item.previewUrl)} otherwise={<T data-testid="no-audio" id="no_audio"/>}>
```

Comment:
Please cross check once why the prettiere is not formating your code on saved and not showing error. Some blocks of code is very difficult to read